lib/onc_certification_g10_test_kit/visual_inspection_and_attestations_group.rb

```diff
@@ -189,6 +189,44 @@ class VisualInspectionAndAttestationsGroup < Inferno::TestGroup
       end
     end
 
+    test do
+      required_suite_options G10Options::SMART_1_REQUIREMENT
+      title 'Health IT developer demonstrated the ability of the Health IT Module / ' \
+            'authorization server to validate token it has issued.'
+      description %(
+        Health IT developer demonstrated the ability of the Health IT Module /
+        authorization server to validate token it has issued
```

Comment:
Could we just add here something like:  "This is a functional requirement that requires manual inspection because SMART App Launch STU1 does not specify a standard approach to token introspection."

Just so we have somewhere "in the code (and UI)" that describes why SMART v1 does not require token introspection as an additional hint why this works as it does.